demos/drawer/temporary-drawer.html

```diff
@@ -0,0 +1,125 @@
+      padding: 0 16px;
+      box-sizing: border-box;
+    }
+
+    @media (min-width: 600px) {
+      .toolbar {
+        height: 64px;
+      }
+    }
+
+    .menu {
+      background: none;
+      border: none;
+      width: 24px;
+      height: 24px;
+      padding: 0;
+      margin: 0;
+      color: #FFF;
+      box-sizing: border-box;
+    }
+
+    main {
+      padding: 16px;
+      box-sizing: border-box;
+    }
+
+    .mdl-temporary-drawer p {
+      padding: 0 16px;
+    }
+    </style>
+  </head>
+  <body>
+    <div class="toolbar mdl-theme--primary-bg mdl-theme--text-primary-on-primary mdl-typography--title mdl-elevation--z4">
+      <button class="menu material-icons">menu</button>
+    </div>
+
+    <aside class="mdl-temporary-drawer mdl-typography">
+      <nav class="mdl-temporary-drawer__drawer">
+        <header class="mdl-temporary-drawer__header">
+          <div class="mdl-temporary-drawer__header-content mdl-theme--primary-bg mdl-theme--text-primary-on-primary">
+            Header here
+          </div>
+        </header>
+        <!--<div class="mdl-temporary-drawer__toolbar-spacer"></div>-->
+        <nav class="mdl-list-group">
+          <div id="icon-with-text-demo" class="mdl-list">
+            <a class="mdl-list-item mdl-temporary-drawer--selected" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">inbox</i>Inbox
+            </a>
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">star</i>Star
+            </a>
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">send</i>Sent Mail
+            </a>
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">drafts</i>Drafts
+            </a>
+          </div>
+
+          <hr class="mdl-list-divider">
+
+          <div class="mdl-list">
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">email</i>All Mail
+            </a>
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">delete</i>Trash
+            </a>
+            <a class="mdl-list-item" href="#">
+              <i class="material-icons mdl-list-item__start-detail" aria-hidden="true">report</i>Spam
+            </a>
+          </div>
+        </nav>
+      </nav>
+    </aside>
+    <main>
+      <h1>Temporary Drawer</h1>
+      <p>Click the menu icon above to open.</p>
+    </main>
+
+    <script src="../assets/material-design-lite.js" charset="utf-8"></script>
+    <script>
+      let drawer = new mdl.TemporaryDrawer(document.querySelector('.mdl-temporary-drawer'));
+      document.querySelector('.menu').addEventListener('click', () => drawer.open = true);
```

Comment:
not sure we can use `let` and `() => {}` here, since they won't work if someone views the demos in IE11 😞